docs/data-types/_index.md

```diff
@@ -20,92 +20,92 @@ If you'd like to try a comprehensive tutorial for each data structure, see their
 
 ### Strings 
 
-[Redis strings](/docs/data-types/strings) are the most basic Redis data type, representing a sequence of bytes.
+[Valkey strings](/docs/data-types/strings) are the most basic Valkey data type, representing a sequence of bytes.
 For more information, see:
 
-* [Overview of Redis strings](/docs/data-types/strings/)
-* [Redis string command reference](/commands/?group=string)
+* [Overview of Valkey strings](/docs/data-types/strings/)
+* [Valkey string command reference](/commands/?group=string)
 
 ### Lists
 
-[Redis lists](/docs/data-types/lists) are lists of strings sorted by insertion order.
+[Valkey lists](/docs/data-types/lists) are lists of strings sorted by insertion order.
 For more information, see:
 
-* [Overview of Redis lists](/docs/data-types/lists/)
-* [Redis list command reference](/commands/?group=list)
+* [Overview of Valkey lists](/docs/data-types/lists/)
+* [Valkey list command reference](/commands/?group=list)
 
 ### Sets
 
-[Redis sets](/docs/data-types/sets) are unordered collections of unique strings that act like the sets from your favorite programming language (for example, [Java HashSets](https://docs.oracle.com/javase/7/docs/api/java/util/HashSet.html), [Python sets](https://docs.python.org/3.10/library/stdtypes.html#set-types-set-frozenset), and so on).
-With a Redis set, you can add, remove, and test for existence in O(1) time (in other words, regardless of the number of set elements).
+[Valkey sets](/docs/data-types/sets) are unordered collections of unique strings that act like the sets from your favorite programming language (for example, [Java HashSets](https://docs.oracle.com/javase/7/docs/api/java/util/HashSet.html), [Python sets](https://docs.python.org/3.10/library/stdtypes.html#set-types-set-frozenset), and so on).
+With a Valkey set, you can add, remove, and test for existence in O(1) time (in other words, regardless of the number of set elements).
 For more information, see:
 
-* [Overview of Redis sets](/docs/data-types/sets/)
-* [Redis set command reference](/commands/?group=set)
+* [Overview of Valkey sets](/docs/data-types/sets/)
+* [Valkey set command reference](/commands/?group=set)
```

Comment:
These datastructures are mentioned in #11.

We should just write "Overview of Sets", "Sets command reference", etc.

Maybe it's better to do this before everything else. Then we can avoid replacing twice ("Redis sets" -> "Valkey sets" -> "Sets"). It's easy to search/replace like this:

    "Redis string" -> "String"
    "Redis set" -> "Set"
    "Redis list" -> "List"
    "Redis hash" -> "Hash"
    "Redis sorted set" -> "Sorted set"
    "Redis stream" -> "Stream"
    "Redis geospatial indexes" -> "Geospatial indexes"
    "Redis bitmaps" -> "Bitmaps"
    "Redis bitfields" -> "Redis bitfields"
    "Redis HyperLogLog" -> "Redis HyperLogLog"

It can actually be done in a separate PR before all other changes. I think it's mostly the files under docs/data-types/ anyway.

Reply:
I made this change in this PR: #29 

@SoulPancake I hope it doesn't cause you trouble.